gradio/components.py

```diff
@@ -1567,9 +1567,6 @@ def postprocess(self, y):
     def deserialize(self, x):
         return processing_utils.decode_base64_to_file(x).name
 
-    def restore_flagged(self, dir, data, encryption_key):
```

Comment:
Why did we remove this function? I remember it being in the output component of `Audio`.

Reply:
Hmmm it looks like we have a discrepancy in how the `inputs.Audio` and `outputs.Audio` are built from flagged. `inputs.Audio` doesn't have this line. Let me look into this some more.

Thanks for catching this

Reply:
What is the latest situation about this issue? I see you removed this function from several components in this [commit](https://github.com/gradio-app/gradio/pull/765/commits/f2a5ecdca6d5ad24b150ff1c154c4cbbf1da1af1). What was its functionality and why remove it?

Reply:
Yeah I'm actually still looking into it. This function is designed to take flagged data and load it back into the interface. It seems like we were handling this differently for inputs and outputs, which was causing some things to break. I can look into it some more when I work on the output tests